README.md

```diff
@@ -42,6 +42,13 @@ In development, we use `lerna` to manage the inter-dependencies of all the packa
  4. Launch the typescript compiler with `yarn compile --watch`
  5. Start ember-cli and the hub node server with `yarn start` and/or run tests with `yarn test`
 
+### Editors
+
+If you use Visual Studio Code and work on the boxel, web-client or ssr-web packages, we recommend the following extensions to make working with glint and gts files better:
+
+* [Glint language server integration for VS Code](https://marketplace.visualstudio.com/items?itemName=typed-ember.glint-vscode)
+* [VSCode extension for glimmer](https://marketplace.visualstudio.com/items?itemName=chiragpat.vscode-glimmer)
```

Comment:
perhaps it might be worth noting that you'll generally see TS errors duplicated if you have the built-in VS Code typescript extension running. When looking at errors you may also see false positives if the built-in VS Code typescript extension is running--so make sure to check the error message to see if it is being reported from glint or tsc--as TSC will not understand glint-isms. (sometimes it might actually be beneficial to turn off the built-in TS extension when working purely in a workspace that is configured for glint, since glint is performing a super set of TS validation)

Reply:
Also, we might want to include instructions on how to turn off the built in typescript extension, because it's not obvious. To find it in the extensions list you need to search for `@builtin typescript`.

Reply:
Good ideas. Added.